.eslintrc.json

```diff
@@ -0,0 +1,55 @@
+{
+  "plugins": [
+    "react"
+  ],
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "env": {
+    "browser": true,
+    "node": false
+  },
+  "extends": [
+    "eslint:recommended"
+  ],
+  "rules": {
+    "brace-style": 1,
+    "comma-dangle": 1,
```

Comment:
It looks like the linter is not supporting comma dangle properly without the full config

"comma-dangle": ["error", "always-multiline"]

https://eslint.org/docs/rules/comma-dangle